.travis.yml

```diff
@@ -57,6 +57,9 @@ matrix:
     - dist: trusty
       env:
         - JOB="3.6, coverage" ENV_FILE="ci/travis-36.yaml" TEST_ARGS="--skip-slow --skip-network" PANDAS_TESTING_MODE="deprecate" COVERAGE=true DOCTEST=true
+    - dist: trusty
```

Comment:
i don't think we need another build

Reply:
you can just put on an existing build

Reply:
That's fair, and I agree we can do that.  I don't have access to my computer ATM (which is why I haven't addressed the comments).